### PR TITLE
Get rid of xorg needle in console test

### PR DESCRIPTION
**Pull request by dzedro**
- Related ticket: https://progress.opensuse.org/issues/60893
- Verification run:
https://openqa.opensuse.org/tests/1110376 TW x11
https://openqa.opensuse.org/tests/1110375 TW wayland
https://openqa.suse.de/tests/3694798 15sp2
https://openqa.suse.de/tests/3694797 15sp1

### CODE REVIEW

**Comment by dzedro** [comment]
> good idea. Can you please also crosscheck a wayland and/or Tumbleweed scenario? Wouldn't need installation runs, e.g. `SCHEDULE=tests/boot/boot_to_desktop,tests/console/consoletest_setup,tests/console/x_vt`

yes, but need HDD_1, added to test runs
I added case for svirt, but I guess it's not needed as the test should check tty, which is not relevant on svirt e.g. s390x

**Comment by dzedro** [comment]
@okurz how does this prove that it's running on correct tty ? https://openqa.opensuse.org/tests/1108936#step/x_vt/3

**Comment by okurz** [comment]
I think wayland does not "run on a tty" so that's the alternative to X.Org

**Comment by dzedro** [comment]
@okurz ok, then this is my code for  it.

File: tests/console/x_vt.pm [hunk 1]
```diff
@@ -15,10 +15,18 @@ use base "consoletest";
 use strict;
 use warnings;
 use testapi;
+use version_utils 'is_tumbleweed';
 
 sub run {
-    assert_script_run('ps -ef | grep bin/X');
-    assert_screen('xorg-tty');
+    if (script_run("ps -ef | grep bin/X | egrep 'tty7|wayland'") == 1) {
+        if (is_tumbleweed && script_run('ps -ef | grep bin/X | grep tty2') == 0) {
```
Comment (okurz) [inline tests/console/x_vt.pm]: Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check. I guess by now also SLE15SP2 behaves like this as there was a recent gnome 3.34 submission

Comment (dzedro) [reply tests/console/x_vt.pm]: I thought the bug is TW live medium specific, I didn't want that some unexpected behaviour will get hidden with generic exception

Comment (okurz) [reply tests/console/x_vt.pm]: I see. It makes sense that this bug was seen there first but I assume by now it is in Tumbleweed and also in SLE15SP2. I assume your tests can show this anyway.

**Comment by dzedro** [comment]
@okurz every SLE test is wayland by default since QEMUVGA=cirrus is not default

**Comment by okurz** [comment]
I doubt that SLE tests are running on wayland. Shouldn't https://openqa.suse.de/tests/3694798#step/x_vt/1 show "Xwayland" instead of "X" as a process then?

**Comment by dzedro** [comment]
sorry, SLED is https://openqa.suse.de/tests/3692329#step/window_system/19

**Comment by dzedro** [comment]
SLES would be too if there were wayland installed by default, here is SLES+ wayland 15sp1 pass.
Had to do it manually and could not switch to tty ...
https://openqa.suse.de/tests/3696922#step/x_vt/1

**Comment by dzedro** [comment]
It will be better when there is such issue on 15sp2 that QA will open bug for it anyway.

**Pull request merged**